test/test_distributions.py

```diff
@@ -745,6 +749,8 @@ def test_cdf_and_icdf(jax_dist, sp_dist, params):
         assert_allclose(d.icdf(d.cdf(samples)), samples, atol=1e-5, rtol=1e-5)
     except NotImplementedError:
         pass
+    except TypeError:
+        pass
```

Comment:
Where might a `TypeError` come from?  Can you confirm that this part actually runs?  I would expect it to fail with `NotImplementedError` because `Poisson.icdf()` is not implemented.

Reply:
I'll skip this part instead. `TypeError` comes from taking grad w.r.t. integer input. We haven't defined a definition for `icdf` of discrete distributions yet.